.github/workflows/pr_test_build.yml

```diff
@@ -179,6 +193,6 @@ jobs:
           token: ${{ secrets.SLACK_APP_TOKEN }}
           path: /opt/android/cake_wallet/build/app/outputs/apk/release/app-release.apk
           channel: ${{ secrets.SLACK_APK_CHANNEL }}
-          title: "${{github.head_ref}}.apk"
-          filename: ${{github.head_ref}}.apk
+          title: "${{env.branch_name}}.apk"
```

Comment:
you are using it with 3 different syntax
- `env.branch_name`
- `BRANCH_NAME`
- `env.BRANCH_NAME`

are all of them working?